### ISSUE TITLE
Add the ability to see the status of each page during translation

### ISSUE THREAD

**Issue by zstix** [newrelic/docs-website#3080]
## Description
At the moment, there is no clear way to discern the status of a page or where it is in the translation process. Some possible states:

* Waiting to be sent to translation vendor
* Currently being translated
* In PR review
* Merged & ready for deployment

This is further complicated by the fact that a page _could_ be in multiple jobs at the same time. Currently our localization manager keeps a (wonderfully detailed) spreadsheet. Once we move to a more relational structure (#3074), we should easily have all the data we need to _show_ the status of each page.

## Strawman Dashboard ™️
```
=== Jobs ===
id | uid | status      | PR(s)      | file(s)
---|-----|-------------|------------|--------
1  | 783 | IN_PROGRESS | #234, #388 | foo.mdx, bar.mdx
2  |     | PENDING     | #234       | baz.mdx

=== Pages ===
id | slug    | status(es)  | job(s)
---|---------|-------------|-------
1  | foo.mdx | IN_PROGRESS | 1
2  | bar.mdx | IN_PROGRESS | 1
3  | baz.mdx | PENDING     | 2
```

## Acceptance Criteria
We should create a view that displays the following information from our database:
* [ ] A list of jobs, their status, and the pages (ideally this also includes the PRs and workflow links)
* [ ] A list of files currently being worked on, the jobs related to them, and their status
* [ ] A list of open PRs to be reviewed

**Comment by jpvajda** [comment]
This is marked as stretch but I assume with our focus on the in product work for o11y packs in the next sprint, this will go to the back. @zstix is that correct?

**Comment by zstix** [comment]
@jpvajda yeah, this is a low priority item. I think @aswanson-nr was considering taking this on as an innovation project.

**Comment by nr-kkenney** [comment]
this should now be resolved with the merge of #3347
